### ISSUE TITLE
Erro na URL de retorno, cartão de débito

### ISSUE THREAD

**Issue by jfoza** [DeveloperCielo/API-3.0-PHP#214]
Estou tendo um erro na URL de retorno para meu site após a compra com cartão de débito. A transação é executada normalmente, vaia para a página de autenticação ou ambiente do banco, mas quando retorna para meu site, dá erro. 
![erro3](https://user-images.githubusercontent.com/58996235/104042634-7e052c80-51b9-11eb-8a7a-88c43811457c.PNG)

**Comment by luizzz** [comment]
Mesmo problema por aqui.
O returnUrl fica nulo, mesmo atribuindo em _setReturnUrl_

**Comment by jfoza** [comment]
Mandei um e-mail para a Cielo descrevendo esse erro, estou aguardando retorno. 
Quando me retornarem vou postar aqui a resposta. 
Isso é coisa deles.

**Comment by luizzz** [comment]
Sim, pois mesmo "forçando" o endereço setando direto, dentro da lib (em /Ecommerce/Payment.php) e desativando o setter, ainda dá problema. Parece que o ambiente da CIELO não está consumindo essa informação.

![image](https://user-images.githubusercontent.com/326745/104380295-8935c080-5509-11eb-8baf-86a353189fa6.png)

Aí o JSON fica preenchido corretamente (não fica com null):

![image](https://user-images.githubusercontent.com/326745/104380069-30febe80-5509-11eb-86ca-699ee44b4245.png)

Mas... ainda dá o erro 404.

**Comment by ozzpy** [comment]
> Mandei um e-mail para a Cielo descrevendo esse erro, estou aguardando retorno.
> Quando me retornarem vou postar aqui a resposta.
> Isso é coisa deles.

eu mandei pedindo suporte a este mesmo SDK, e me disseram que tem que seguir a documentacao oficial, ou seja esse SDK nao tem suporte.
